### DISCUSSION TITLE
Are there any plans to introduce support for bazel?

### DISCUSSION THREAD

**Discussion by ylmzkaan** [OpenDDS/OpenDDS#3707] [Ideas]

**Comment by simpsont-oci** [comment]
As far as I know, there are no plans to support any build systems beyond MPC and CMake. Do you have a strong need for bazel support?

**Reply by ylmzkaan** [reply]
Not at the moment. I am writing a bazel wrapper around make as a workaround. If I can make it work then I'll share it here and see if it is worth creating a pull request. Otherwise, I'll have too see what other option are available.
